.github/release.yml

```diff
@@ -12,3 +12,6 @@ changelog:
     - title: 🧑‍💻 Dependencies
       labels:
         - dependencies
+    - title: 💬 Localisation
```

Comment:
Due to the fact that the IT world largely comes from America, it is more common to see "localization". Although your version is also correct, but for British English.

Reply:
Considering things like the contributing guidelines are written using American English, for the sake of consistency, I have updated the spelling.